virttest/qemu_vm.py

```diff
@@ -3656,6 +3658,8 @@ def activate_netdev(self, nic_index_or_name):
                 utils_net.add_to_bridge(nic.ifname, nic.netdst)
         elif nic.nettype == 'user':
             attach_cmd += " user,id=%s" % nic.device_id
+        elif nic.nettype == 'socket':
+            attach_cmd += " socket,id=%s" % nic.device_id
```

Comment:
I think we should add `listen` `connect` params here, since they are essential element for socket, not extra.